README.md

```diff
@@ -32,3 +32,10 @@ channel (used for programm logs) of your Discord server. The format should look
     LOG_CHANNEL_ID=IDHERE
 
 You can then compile the project using `mvn clean install`. Afterwards run `java -jar target/Zuap-jar-with-dependencies.jar` in a terminal to execute the program.
+
+### WGZimmer
+
+WGZimmer uses reCAPTCHA to prevent bots from scraping their website. To bypass this, selenium is used.
+You need to install firefox and the geckodriver for selenium to work.
+Even then, reCAPTCHA gets triggered in some cases (i.e. it works on my desktop, but not on my server).
+If you encounter this problem, you can add `NO_WGZIMMER=true` to your `.env` file to disable WGZimmer scraping.
```

Comment:
Since there are additional requirements for using WGZimmer, I suggest we make the usage of that website opt-in rather than opt-out (i.e. the option should be `ENABLE_WGZIMMER` instead)

Reply:
done